.github/workflows/template-cleanup.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           GROUP="com.github.$SAFE_ORG"
 
           # Replace placeholders
-          sed -i "s/%NAME%/$NAME/g" build.sbt build.sc README.md
+          sed -i "s/%NAME%/$NAME/g" build.sbt build.sc README.md src/test/scala/gcd
```

Comment:
@jackkoenig template creation fails for me with "sed: couldn't edit src/test/scala/gcd: not a regular file".
Maybe use 'src/test/scala/gcd/*'?

Reply:
Oops, my fault, you are right.

Reply:
Good catch, thanks @towoe! I'll push a fix.